.github/renovate.json

```diff
@@ -0,0 +1,208 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "enabledManagers": [
+    "npm"
+  ],
+  "force": {
+    "constraints": {
+      "node": "< 15.0.0",
+      "npm": "< 7.0.0"
+    }
+  },
+  "pruneStaleBranches": false,
+  "rangeStrategy": "bump",
+  "commitMessagePrefix": "patch:",
+  "commitBody": "Update {{depName}} from {{currentVersion}} to {{newVersion}}\n\nChange-type: patch",
+  "prHourlyLimit": 0,
+  "labels": [
+    "dependencies"
+  ],
+  "ignoreDeps": [
+    "node"
+  ],
+  "packageRules": [
+    {
+      "groupName": "storybook-non-major",
+      "matchPackagePrefixes": ["@storybook"],
+      "commitBody": "Change-type: patch",
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "automerge": true
+    },
+    {
+      "groupName": "storybook-major",
+      "matchPackagePrefixes": ["@storybook"],
+      "commitBody": "Change-type: patch",
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "automerge": false
+    },
+    {
+      "groupName": "types",
+      "matchPackagePrefixes": ["@types"],
+      "commitBody": "Change-type: patch",
+      "matchUpdateTypes": [
+        "major",
+        "minor",
+        "patch"
+      ],
+      "automerge": true
+    },
+    {
+      "groupName": "fontawesome",
+      "matchPackagePrefixes": ["@fortawesome"],
```

Comment:
Typo: `@fortawesome` => `@fontawesome`

Reply:
`@fortawesome` is the organization name `fontawesome` is the library name, I thought to group using the library name, but I can change it if it's confusing  (ref: https://github.com/balena-io-modules/rendition/blob/c2b38484cc9ea9c0290526cff8364711ae5a58e5/package.json#L89 )